src/test/java/org/folio/edge/controller/InnReachProxyControllerTest.java

```diff
@@ -98,4 +107,72 @@ void shouldUpdateRequestHeaders() {
     );
   }
 
+  @Test
+  void shouldHandleCommonErrors() {
+    var httpHeaders = createInnReachHttpHeaders();
+    httpHeaders.set(ACCEPT_ENCODING, "gzip");
+    httpHeaders.set(AUTHORIZATION, AUTH_TOKEN_VALUE);
+
+    InnReachResponseDTO innReachResponseDTO = new InnReachResponseDTO();
+    innReachResponseDTO.status("failed");
+    ObjectMapper obj = new ObjectMapper();
+    String jsonStr = null;
+    try {
+      jsonStr = obj.writeValueAsString(innReachResponseDTO);
+    }
+    catch (IOException e) {
+      e.printStackTrace();
+    }
```

Comment:
save a response as json file in src/test/resources/... then read it with `TestUtil.readFileContentAsString`. Add some "reason" value as well

Reply:
Fixed